### DISCUSSION TITLE
Extract people occupations from a doc

### DISCUSSION THREAD

**Discussion by YannBrrd** [explosion/spaCy#11023] [Help: Coding & Implementations] [answered]
Hi,

I was wondering how to extract people occupation from a doc.

Assuming I have this phrase as an input
```text
BAA's chairman Sir Nigel Rudd and chief executive Colin Matthews, who appeared before the committee, were criticised for being "unhelpful and ill-prepared" and not being able to provide any "satisfactory explanation" about why the chaos had happened.
```

 How to get an array  with : 
| Name | Title |
|-------- | ------ |
| Sir Nigel Rudd | BAA's chairman |
| Collin Matthews | Chief executive |

Thanks

**Comment by polm** [comment, marked as answer]
In this case I would suggest you first train an NER model for occupations and then use the [relation extraction component](https://github.com/explosion/projects/tree/v3/tutorials/rel_component) to match PERSON to OCCUPATION entities. 

Note that putting it in a table is a minor step, but may have issues, because there's no guarantee everyone has exactly one occupation in a sentence. So keep that in mind.
